contents/blog/give-back-friday-2021.md

```diff
@@ -0,0 +1,73 @@
+| [Rrweb](https://github.com/rrweb-io/rrweb)                               | An open-source web session replay library.                            | Rrweb powers [our session recording features](https://posthog.com/docs/user-guides/recordings) |
+| [Graphile/Worker](https://github.com/graphile/worker)                    | A high performance Node.js/PostgreSQL job queue.                      | Graphile/Worker powers logic for our plugin server.                                            |
+| [Django REST framework](https://github.com/encode/django-rest-framework) | Open-source web APIs for Django.                                      | This powers the PostHog REST APIs.                                                             |
+| [KeePass](https://keepass.info/donate.html)                              | An open-source multi-platform password manager.                       | Keepass is used regularly by our team and was suggested by Joe Martin.                         |
+| [Piscina](https://github.com/piscinajs/piscina)                          | A fast, efficient Node.js Worker Thread Pool implementation.          | We run a fork of Piscina which powers activities across PostHog.                               |
+| [All-contributors](https://github.com/all-contributors/all-contributors) | A tool to recognize community contributions to open-source projects.  | We run a form of All-contributors to power our contributions bot.                              |
+| [Alex](https://github.com/all-contributors/all-contributors)             | A tool to detect insensitive or inconsiderate writing.                | Alex is used by our team and was suggested by Paul D'Ambra.                                    |
+| [Ant-design](https://github.com/all-contributors/all-contributors)       | An open-source UI design language and React UI library.               | Ant-design is used heavily across both our website and product.                                |
+
+## How does this work?
+
+That’s easy. Between now and 00:01 BST, 30 November simply follow these steps:
+
+1. Login to GitHub and go to [the PostHog repo](https://github.com/PostHog/posthog).
+2. Click the :star: icon. Hint: it's in the top right corner.
+3. That’s it. We'll donate on your behalf!
+
+On the morning of Tuesday, 30 November we’ll count up how many stars we’ve gained and contact the projects above to make the donations. We'll quickly verify the stars are legitimate (see 'Small Print' below).
+
+## Why are we doing this?
+
+PostHog is an open-source project and our team is all individually proud to be part of the open-source community. We wanted to create a festive activity but, because most of our paying users are other businesses, we didn’t want to simply slash prices. We wanted to truly celebrate and support the open-source community in a meaningful way.
```

Comment:
I didn't edit this bit directly because I wasn't quite sure of the right approach, but the sentence _'We wanted to create a festive activity but, because most of our paying users are other businesses, we didn’t want to simply slash prices.'_ is a bit weird - it's not really a festive activity, and the reason for not slashing prices isn't quite right. 

Maybe instead we can say something like:

PostHog is an open-source project and our team is all individually proud to be part of the open-source community. We wanted to take this weekend as an opportunity to truly celebrate and support the open-source community in a meaningful way. We don't do short-lived price cuts, as we don't believe it's the right way to build trust with our customers and a sustainable business in the long run.